pyanaconda/modules/storage/installation.py

```diff
@@ -327,6 +334,29 @@ def _write_escrow_packets(self, storage, sysroot):
 
         log.debug("escrow: write_escrow_packets done")
 
+    def _write_lvm_devices_file(self, storage, sysroot):
```

Comment:
It would be great to have some unit tests (or kickstart tests) for this functionality. If you make this method static, you can call it directly from the tests.